src/WormholeReceiver.sol

```diff
@@ -38,7 +38,7 @@ abstract contract WormholeReceiver is Ownable, WormholeBase {
   /// @param sourceChain Chain that the delivery was requested from.
   /// @param deliveryHash Unique identifier of this delivery request.
   function receiveWormholeMessages(
-    bytes memory payload,
+    bytes calldata payload,
```

Comment:
Should we use internal methods for these modifiers like number 8 in 0xMacro

Reply:
number 8 relates more to deploy cost rather than gas use. since we aren't near the bytecode limit, I'm more inclined to leave the modifiers as they are, but will defer to you; lmk!

Reply:
Sounds good to me